docs/data-abstraction/syllabus-data-abstraction.md

```diff
@@ -49,43 +46,58 @@ presentations.
 - Prerequisite: CMPSC 100 or permission of the instructor.
 - Distribution Requirements: QR, SP.
 
+???+ note "What do these distribution requirements mean?"
+
```

Comment:
Awesome! I agree that it is a good idea to include these descriptions in the syllabus. I am going to include them in this fashion as well for the CMPSC 203 syllabus that I am creating!